tools/lsmcli/cmdline.py

```diff
@@ -798,6 +859,12 @@ def _get_item(l, the_id, friendly_name='item', raise_error=True):
     ['vilc', 'volume-ident-led-clear'],
     ['srcpu', 'system-read-cache-pct-update'],
     ['pmi', 'pool-member-info'],
+    ['ldl', 'local-disk-list'],
+    ['lb', 'list --type batteries'],
+    ['vci', 'volume-cache-info'],
+    ['vcp', 'volume-phy-disk-cache-set'],
```

Comment:
These last three aliases confuse me. So far, when I create new ones, I've been using the first letter from every work in the operation (see line 860). They're longer, sure, but the user doesn't need to guess which letter actually made it into the alias (here, it's pretty arbitrary that 'volume-phy-disk-cache-set' ended up as vcp. You took the first letters from volume, cache, and phy, but put them in a different order than they exist in the extended command name...as a user, that would frustrate me. I would suggest 'vpdcs' for this specific one, and continue to use the first letter from each word in the extended command for these aliases.